src/batch.rs

```diff
@@ -13,10 +13,7 @@ use std::convert::TryFrom;
 use jubjub::*;
 use rand_core::{CryptoRng, RngCore};
 
-use crate::{
-    scalar_mul::VartimeMultiscalarMul, Error, HStar, Scalar, SigType, Signature, VerificationKey,
-    VerificationKeyBytes,
-};
+use crate::{private::Sealed, scalar_mul::VartimeMultiscalarMul, *};
```

Comment:
Does this actually use Sealed anywhere?

Reply:
I think this is yet more detritus from my failed experiment 👍

Reply:
EDIT: I need to have this in scope to call `basepoint()` on the different `SigType`s.